testscripts/testrunner/run_test.test.txt

```diff
@@ -0,0 +1 @@
+exec devbox run run_test
```

Comment:
Will we be adding specific testscripts for other examples? If the testscript is just running `devbox run run_test` why not do that directly in a regular test?

Reply:
The motivation is ensuring isolation is better...